packages/ui/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack, VStack } from '../Stack';
+import { Checkbox, CheckboxProps } from '.';
+
+const UnCheckedCb = (props: Omit<CheckboxProps, 'onChange' | 'checked'>) => (
+  <Checkbox {...props} onChange={() => {}} checked={false} />
+);
+const CheckedCb = (props: Omit<CheckboxProps, 'onChange' | 'checked'>) => (
+  <Checkbox {...props} onChange={() => {}} checked />
+);
+
+const meta: Meta = {
+  title: 'Components / Checkbox',
+  component: Checkbox,
+};
+
+export default meta;
+
+const Template: Story<CheckboxProps> = (args) => <Checkbox {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  children: 'Checkbox text',
+  size: 'md',
+  disabled: false,
+  checked: false,
+  name: 'terms',
+  value: 'agree',
+};
+
+export const Checkboxes = () => (
```

Comment:
This solution looks inconsistent when you look at other components in our codebase. It could be better if you look at the [Radio PR](https://github.com/CodersCrew/coderscamp/pull/46/files) and create checkbox stories in a similar way